common/views/components/OpeningHours/OpeningHours.js

```diff
@@ -41,7 +41,7 @@ class OpeningHours extends Component<Props, State> {
       <Fragment>
         {!placesOpeningHours &&
           <p className={spacing({s: 2}, {margin: ['bottom']})}>
-            <a className={font({s: 'HNL6', m: 'HNL5'})} href="https://wellcomecollection.org/info/opening-times">
+            <a className={font({s: 'HNL6', m: 'HNL5'})} href='https://wellcomecollection.org/info/opening-times'>
```

Comment:
What links are these?

Reply:
Ah, good question (I think this change must've been an autolint correction).